test/unit/util/obfuscate-sql.test.js

```diff
@@ -4,46 +4,43 @@
  */
 
 'use strict'
-
-const tap = require('tap')
+const assert = require('node:assert')
+const test = require('node:test')
 const tests = require('../../lib/cross_agent_tests/sql_obfuscation/sql_obfuscation')
 const obfuscate = require('../../../lib/util/sql/obfuscate')
 
-tap.test('sql obfuscation', (t) => {
-  tests.forEach((test) => {
-    t.test(test.name, (t) => {
-      for (let i = 0; i < test.dialects.length; ++i) {
-        runTest(t, test, test.dialects[i])
-      }
-      t.end()
-    })
-  })
-
-  function runTest(t, test, dialect) {
-    t.comment(dialect)
-    const obfuscated = obfuscate(test.sql, dialect)
-    if (test.obfuscated.length === 1) {
-      t.equal(obfuscated, test.obfuscated[0])
-    } else {
-      t.ok(test.obfuscated.includes(obfuscated))
-    }
+function runTest(t, testCase, dialect) {
+  t.diagnostic(dialect)
```

Comment:
Unfortunately, this isn't going to print inline with the actual test in the output. Diagnostic messages get collected and reported after all tests are done 😢

Reply:
they seem to post inline

```sh
  ℹ postgres
  ℹ cassandra
  ✔ uuid_with_braces (0.01925ms)
  ℹ postgres
  ✔ uuid_no_dashes (0.01825ms)
  ℹ postgres
  ✔ uuid_random_dashes (0.017791ms)
  ℹ postgres
  ✔ booleans (0.019708ms)
  ℹ mysql
  ℹ postgres
  ℹ cassandra
  ℹ sqlite
  ✔ in_clause_digits (0.021291ms)
  ℹ mysql
  ℹ postgres
  ℹ oracle
  ℹ cassandra
  ℹ mssql
  ✔ in_clause_strings (0.028792ms)
  ℹ mysql
  ℹ postgres
  ℹ oracle
  ℹ cassandra
  ℹ mssql
  ✔ should handle line endings (0.045709ms)
```